aip/general/0162.md

```diff
@@ -234,17 +149,12 @@ there are two potential variants:
 If a child resource is a child of a single revision, the child resource's name
 **must** always explicitly include the parent's revision ID:
 
-    publishers/123/books/les-miserables@c7cfa2a8/pages/42
-
-In `List` requests for such resources, the service **should** default to the
-latest revision of the parent if the user does not specify one, but **must**
-explicitly include the parent's revision ID in the `name` field of resources in
-the response.
+    publishers/123/books/les-miserables/revisions/c7cfa2a8/pages/42
```

Comment:
I think we need more detail about the messages for child resources. In the previous scheme, we didn't have different messages for revisions and non-revisions, because they used the same resource name. Now we've got different messages for the top-level resources - do we have that for child resources as well? If not, is there any way of knowing which top-level revision a child resource was fetched from?